internal/gengapic/testdata/empty_client_init.want

```diff
@@ -100,14 +100,38 @@ func (c *gRPCClient) Close() error {
 // Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.
 type restClient struct {
 	host string
+	httpClient http.Client
 }
 
 func (c *restClient) Zip(ctx context.Context, req *mypackagepb.Bar, opts ...gax.CallOption) (*mypackagepb.Foo, error) {
-	m := jsonpb.Marshaler{}
-	if jsonReq, err := m.MarshalToString(req); err != nil {
+	// The default (false) for the other options are fine.
+	marshaler := protojson.MarshalOptions{AllowPartial: true, Multiline: true, EmitUnpopulated: true}
+	if jsonReq, err := marshaler.Marshal(req); err != nil {
 		return nil, err
+	}
+
+	url := fmt.Sprintf("https://%s/zip", c.host)
+	httpReq, err := http.NewRequestWithContext(ctx, "GET", url, bytes.NewReader(jsonReq))
+	if err != nil {
+		return nil, err
+	}
 
+	httpRsp, err := client.Do(httpReq)
+	defer httpRsp.Body.Close()
+	if err != nil{
+		return nil, err
+	} else if httpRsp.StatusCode >= 400 {
+		return nil, errors.E(nil, httpRsp.Status)
```

Comment:
I am not sure what errors.E is, but I don't think we want this dependency? We should process errors similarily if not exactly how it is done in the apiary libraries. There is some special logic to parse known proto struct errors there. Example: https://github.com/googleapis/google-api-go-client/blob/v0.46.0/googleapi/googleapi.go#L130